features.json

```diff
@@ -82,11 +82,11 @@
 				"bulkMemory": true,
 				"multiValue": true,
 				"mutableGlobals": true,
-				"referenceTypes": "Nightly",
+				"referenceTypes": true,
 				"saturatedFloatToInt": true,
 				"signExtensions": true,
-				"simd": "Beta 79, x86 and x86-64 only",
-				"threads": "Beta 79"
+				"simd": "x86 and x86-64 only",
```

Comment:
Oh nice, is SIMD enabled by default on x86/x86_64 now?

Reply:
Yeah it was on Beta too, so looks that way!

Reply:
Odd, the Firefox 79 release notes don't mention it,

https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/79

Perhaps an oversight in the notes? Or maybe it wasn't flipped on?

Reply:
SIMD is Nightly-only.  There's work ongoing to let it ride the train to Release, but preffed off (since Phase 3).